### PR TITLE
Added new Board and Move class.  

### PR DESCRIPTION
**Pull request by jordan-elley-myob**

### CODE REVIEW

File: TicTacToe.Test/MoveShould.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+using System;
+using Xunit;
+
+namespace TicTacToe.Test
+{
+    public class MoveShould
+    {
+        [Fact]
+        public void HaveAnXValue()
```
Comment (karenmyob) [inline TicTacToe.Test/MoveShould.cs]: Naming: HaveAnXAndYValue()

File: TicTacToe.Test/WinCheckerShould.cs [hunk 2]
```diff
@@ -0,0 +1,79 @@
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace TicTacToe.Test
+{
+    public class WinCheckerShould
```
Comment (karenmyob) [inline TicTacToe.Test/WinCheckerShould.cs]: Let's remove this from this PR as discussed

File: TicTacToe.Tests/UnitTest1.cs [hunk 3]
```diff
@@ -3,11 +3,21 @@
 namespace TicTacToe.Tests
 {
     [TestClass]
-    public class UnitTest1
+    public class BoardShould
```
Comment (karenmyob) [inline TicTacToe.Tests/UnitTest1.cs]: Delete this class

File: TicTacToe/Board.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
+using System.Collections.Generic;
+
+namespace TicTacToe
+{
+    public class Board
```
Comment (karenmyob) [inline TicTacToe/Board.cs]: For this class, can you think of a design which isn't concerned about players?
ie. A board which simply holds tokens which have been played and their position?

Comment (jordan-elley-myob) [reply TicTacToe/Board.cs]: What if I called the 2 lists: xTokens and yTokens.  Would that then make it a board concern

**Review by karenmyob — APPROVED** [review]

**Pull request merged**